components/API/core/apis_impl.bal

```diff
@@ -0,0 +1,118 @@
+// ------------------------------------------------------------------------
+//
+// Copyright 2019 WSO2, Inc. (http://wso2.com)
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License
+//
+// ------------------------------------------------------------------------
+import ballerina/log;
+import cellery_hub_api/db;
+
+public function getOrg (http:Request _getOrgReq) returns http:Response{
+    var params = _getOrgReq.getQueryParams();
+    http:Response _getOrgRes = new;
+    if (params.hasKey(ORG_NAME)){
+        string orgName = <string>params[ORG_NAME];
+
+        json|error res = db:getOrganization(orgName);
+
+        if (res is json){
+            _getOrgRes.statusCode = SUCCESS_STATUSCODE; 
+            _getOrgRes.setJsonPayload(untaint res.organizationresponseList);
+            log:printInfo("Successfully fetched organization " +orgName+ " from REGISTRY_ORGANIZATION" );        
+            return _getOrgRes;
+        }
+        else{      
+            string errMsg = "Unexpected error occured when fetching data from REGISTRY_ORGANIZATION";
+            log:printError(errMsg , err = res);        
+            return errorResponse(INTERNAL_ERROR_STATUSCODE, errMsg, untaint <string>res.detail().message); 
+        }        
+    }
+    else{
+        string errMsg = "getOrg faild : Organization name is not specified";
+        log:printError(errMsg);
+        return errorResponse(METHOD_NOT_ALLOWD_STATUSCODE, errMsg, "");
+    } 
+}
+
+public function addOrg (http:Request _addOrgReq, gen:organizationRequest _addOrgBody) returns http:Response {
+    // stub code - fill as necessary
+    http:Response _addOrgRes = new;
+    string _addOrgPayload = "Sample addOrg Response";
+    _addOrgRes.setTextPayload(_addOrgPayload);
+
+	return _addOrgRes;
```

Comment:
Although this is a generated file, since we are editing it, lets format the code properly.